drivers/sensor/isl29125/isl29125.c

```diff
@@ -20,7 +20,8 @@ static u8_t isl29125_read8(struct device *i2c_dev, u8_t reg)
 	u8_t buf;
 
 	if (i2c_reg_read_byte(i2c_dev, ISL29125_I2C_ADDR, reg, &buf) < 0) {
-		LOG_ERR("Error reading register %d at Addr:0x%x", reg, ISL29125_I2C_ADDR);
+		LOG_ERR("Error reading register %d at Addr:0x%x", reg,
+			ISL29125_I2C_ADDR);
```

Comment:
Please amend previous commits to make all of the reformatting fixes instead of making a new commit